platform/vechain/api.go

```diff
@@ -311,16 +321,16 @@ func NormalizeTokenTransaction(t *NativeTransaction) (tx blockatlas.Tx, ok bool)
 		},
 	}, true
 }
+
 // NormalizeTransaction converts a VeChain transaction into the generic model
-func NormalizeTransaction(t *NativeTransaction) (tx blockatlas.Tx, ok bool) {
+func NormalizeTransaction(t *NativeTransaction, outputIndex int, transferIndex int) (tx blockatlas.Tx, ok bool) {
```

Comment:
I would avoid adding `outputIndex int, transferIndex int`.

Maybe better to return `tx []blockatlas.Tx` instead?